mlflow/server/js/src/experiment-tracking/components/ExperimentView.js

```diff
@@ -21,6 +21,9 @@ import {
   LIFECYCLE_FILTER,
   MAX_DETECT_NEW_RUNS_RESULTS,
   MODEL_VERSION_FILTER,
+  DEFAULT_ORDER_BY_KEY,
+  DEFAULT_ORDER_BY_ASC,
+  DEFAULT_START_TIME,
 } from './ExperimentPage';
```

Comment:
Interesting. Never knew we had a cyclic deps. Like ExperimentPage depends on ExperimentView and ExperimentView depends on ExperimentPage. 
Not sure if we can solve this as part of this PR. But something we should plan to refactor for sure. 

cc @xanderwebs

Reply:
I've tried to get rid of the cyclic dependencies by refactoring the constants used in the Experiments UI into constants.js. Curious to hear your thoughts on it

Reply:
Yup that works. Thank you so much :)